tensorly/contrib/decomposition/_tt_cross.py

```diff
@@ -105,16 +103,12 @@ def tensor_train_cross(input_tensor, rank, tol=1e-4, n_iter_max=100, random_stat
     # Initialize rank
     if rank[0] != 1:
         print(
-            "Provided rank[0] == {} but boundary conditions dictate rank[0] == rank[-1] == 1: setting rank[0] to 1.".format(
-                rank[0]
-            )
+            f"Provided {rank[0] = } but boundary conditions dictate rank[0] == rank[-1] == 1: setting rank[0] to 1."
```

Comment:
Changed to support Python 3.7 and shortened line

```suggestion
            f"Provided rank[0] = {rank[0]} but boundary conditions dictate "
            + "rank[0] == rank[-1] == 1: setting rank[0] to 1."
```